docker/spack-stack/create_dockerfile.sh

```diff
@@ -1,108 +1,31 @@
 #!/bin/bash
 
 # Clone spack-stack
-spack_stack_version=1.6.0
+spack_stack_version=1.7.0
 git clone -b $spack_stack_version --recursive https://github.com/JCSDA/spack-stack.git
 
 # Create the spack-stack spack.yaml file
 pushd spack-stack
 . ./setup.sh
 spack stack create ctr --container=docker-ubuntu-gcc-openmpi --specs=jedi-ci
 
-# Modify spack.yaml to turn off openmpi static libraries
-cd envs/docker-ubuntu-gcc-openmpi
-perl -p -i -e "s/variants: \+internal-hwloc \+two_level_namespace/variants: \+internal-hwloc \+two_level_namespace \~static/g" spack.yaml
-
-# Modify spack.yaml to increase parallelism
-perl -p -i -e "s/build_jobs: 2/build_jobs: 8/g" spack.yaml
-
-# Modify spack.yaml to add flux-core and flux-sched
-perl -p -i -e "s/fms:/flux-core:\n      version: [0.58.0]\n    fms:/g" spack.yaml
-perl -p -i -e "s/fms:/flux-sched:\n      version: [0.32.0]\n    fms:/g" spack.yaml
-perl -p -i -e "s/fms\@/flux-core\@0.58.0\n  - fms\@/g" spack.yaml
-perl -p -i -e "s/fms\@/flux-sched\@0.32.0\n  - fms\@/g" spack.yaml
-
-# Modify spack.yaml to adjust boost variants for flux compatibility
-export boost_variants=$(
-cat<<'BOOST_EOF'
-      variants: ~atomic +chrono ~clanglibcpp +container ~context ~contract ~coroutine +date_time
-        ~debug +exception ~fiber +filesystem +graph ~graph_parallel ~icu ~iostreams ~json
-        ~locale ~log ~math ~mpi +multithreaded ~nowide ~numpy +pic +program_options +python
-        ~random +regex +serialization +shared ~signals ~singlethreaded ~stacktrace +system
-        ~taggedlayout +test +thread +timer ~type_erasure ~versionedlayout ~wave cxxstd=17
-        visibility=hidden
-BOOST_EOF
-)
-perl -i -p0e 's/      variants\:.*visibility=hidden/$ENV{boost_variants}/s' spack.yaml
-
-# Modify spack.yaml to install py-pytest, py-flake8, and py-pytest-flake8
-perl -p -i -e "s/# To avoid/py-flake8:\n      version: [6.1.0]\n    # To avoid/g" spack.yaml
-perl -p -i -e "s/# To avoid/py-pytest-flake8:\n      version: [0.8.1]\n    # To avoid/g" spack.yaml
-perl -p -i -e "s/py-pyyaml\@6.0/py-pytest\@7.3.2\n  - py-pyyaml\@6.0/g" spack.yaml
-perl -p -i -e "s/py-pyyaml\@6.0/py-flake8\@6.1.0\n  - py-pyyaml\@6.0/g" spack.yaml
-perl -p -i -e "s/py-pyyaml\@6.0/py-pytest-flake8\@0.8.1\n  - py-pyyaml\@6.0/g" spack.yaml
-
 # Create the spack-stack Dockerfile
+pushd envs/docker-ubuntu-gcc-openmpi
 spack containerize > ../../../Dockerfile
 
 # If necessary move spack-ext-* for use in COPY during container build
 rm -rf ../../../spack-ext-*
-find . -type d -name 'spack-ext-*' -exec mv {} ../../../ \;
+mv spack-ext-* ../../../
 
 # Remove unneeded spack-stack install
 popd
+popd
 rm -rf spack-stack
```

Comment:
```suggestion
popd
rm -rf spack-stack
```

Duplicate `popd`

Reply:
It's not a duplicate.  We need to popd twice to undo two `cd` commands.